Dockerfile.prod

```diff
@@ -0,0 +1,28 @@
+FROM python:3.12.7-alpine3.20@sha256:edd1d8559c585e1e9a9b79de44ac27f8ac32cb0c7323e112ae6870ceeecd8dbf AS builder
+
+COPY requirements.txt ./
+
+RUN set -eux; \
+    \
+    # Install pip dependencies \
+    python3 -m pip install --no-cache-dir -r requirements.txt; 
+
+FROM python:3.12.7-alpine3.20@sha256:edd1d8559c585e1e9a9b79de44ac27f8ac32cb0c7323e112ae6870ceeecd8dbf
+
+WORKDIR /object-storage-api-run
+
+COPY --from=builder /usr/local/lib/python3.12/site-packages /usr/local/lib/python3.12/site-packages
+COPY --from=builder /usr/local/bin /usr/local/bin
+
+COPY README.md ./
+COPY object_storage_api/ object_storage_api/
+
+RUN set -eux; \
+    \
+    # Create loging.ini from its .example file \
+    cp object_storage_api/logging.example.ini object_storage_api/logging.ini;
+
+USER nobody
```

Comment:
Wasn't aware of this user. We just create a non-root user and group in other projects. I will need to look into this a bit more because if we can just use `nobody` then it would save us from creating non-root users. Let me know if you did any research on this and what you find out.